Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -35220,4 +35229,1126 @@
+        <Model value="MedicHealModel"/>
+        <Host Subject="_Selectable"/>
+        <HostSiteOps Ops="SOpAttachOverhead"/>
+    </CActorModel>
+    <CActorSplat id="AP_StukovInfestedSiegeTankDeepTunnelSplatFinish" parent="BurrowedSplatBase">
+        <On Terms="Effect.AP_StukovInfestedSiegeTankDeepTunnelTeleport.Stop" Send="Create"/>
+        <On Terms="ActorCreation" Send="RefClear ::Host"/>
+        <On Terms="ActorCreation" Send="TimerSet 4.000000,2.000000 DelayDestroy"/>
+        <On Terms="TimerExpired; TimerName DelayDestroy" Send="Destroy"/>
+        <HostSiteOps Ops="SOpAttachOriginStationary SOpShadow"/>
+        <Scale value="3.000000"/>
+        <Model value="AP_StukovInfestedSiegeTankDeepTunnelBurrowedSplatModel"/>
+    </CActorSplat>
+    <CActorSplat id="AP_StukovInfestedSiegeTankDeepTunnelSplatStart" parent="BurrowedSplatBase">
+        <On Terms="Abil.AP_StukovInfestedSiegeTankDeepTunnel.SourceCastStart" Send="Create"/>
+        <On Terms="ActorCreation" Send="RefClear ::Host"/>
+        <On Terms="ActorCreation" Send="TimerSet 4.000000,2.000000 DelayDestroy"/>
+        <On Terms="TimerExpired; TimerName DelayDestroy" Send="Destroy"/>
+        <HostSiteOps Ops="SOpAttachOriginStationary SOpShadow"/>
+        <Scale value="3.000000"/>
+        <Model value="AP_StukovInfestedSiegeTankDeepTunnelBurrowedSplatModel"/>
+    </CActorSplat>
+    <CActorUnit id="AP_StukovInfestedSiegeTankDeepTunnelPlacementUnit" parent="GenericUnitMinimal" unitName="AP_StukovInfestedSiegeTankDeepTunnelPlacementUnit">
+        <Model value="Invisible"/>
+        <UnitFlags index="ShowBlobShadow" value="0"/>
+    </CActorUnit>
+    <CActorModel id="AP_StukovInfestedSiegeTankDeepTunnelUnburrow" parent="ModelAnimationStyleOneShot">
+        <On Terms="Effect.AP_StukovInfestedSiegeTankDeepTunnelTeleport.Start; At Caster" Send="Create"/>
+        <Model value="AP_StukovInfestedSiegeTankDeepTunnelUnburrowDustModel"/>
+    </CActorModel>
+    <CActorModel id="AP_StukovInfestedSiegeTankDeepTunnelBurrow" parent="ModelAnimationStyleOneShot">
+        <Host Subject="_Selectable"/>
+        <HostSiteOps Ops="SOpAttachOriginStationary SOpShadow"/>
+        <Model value="AP_StukovInfestedSiegeTankDeepTunnelBurrowDustModel"/>
+        <On Terms="Abil.AP_StukovInfestedSiegeTankDeepTunnel.SourceCastStop" Send="Create"/>
+    </CActorModel>
+    <CActorSound id="AP_Spell_StukovInfestedSiegeTank_DeepTunnel_Burrow" parent="SoundOneShot"/>
+    <CActorSound id="AP_Spell_StukovInfestedSiegeTank_DeepTunnel_Unburrow" parent="SoundOneShot"/>
+    <CActorRange id="AP_StukovInfestedSiegeTankSiegedRange">
+        <Weapon value="AP_StukovInfestedSiegeTank"/>
+        <On Terms="SelectionLocalUpdate.AP_StukovInfestedSiegeTank.Start; ValidateUnit AP_StukovInfestedSiegeTankAmmoGT0" Send="Create"/>
+        <On Terms="Behavior.AP_StukovInfestedSiegeTankAmmo.On; ValidateUnitType IsSISiegeTankSieged" Send="Create"/>

```

Comment:
![Snímek obrazovky pořízený 2024-09-09 22-17-45](https://github.com/user-attachments/assets/236e0515-3341-42e8-9b77-d20a6b1d4616)
Missing `AP_` Prefix in Validator